src/index.ts

```diff
@@ -150,3 +152,54 @@ export const major = (version: string | null): number | null => {
 
 	return null;
 };
+
+/**
+ * @summary Check if a version is greater than or equal to another
+ * @name gte
+ * @public
+ * @function
+ *
+ * @description Returns true if versionA is greater than or equal to versionB.
+ * Valid semver versions are always weighted above non semver strings.
+ * Non-semver strings are compared alphabetically.
+ *
+ * @param {string|null} versionA - The version string to compare against
+ * @param {string|null} versionB - The version string to compare to versionA
+ *
+ * @returns {boolean} - true if versionA is greater than or equal to versionB, otherwise false.
+ *
+ * @example
+ * resinSemver.gte('2.0.5', '1.16.0'); //--> true
+ *
+ * resinSemver.gte('Resin OS 2.0.5', 'Resin OS 2.0.2+rev2'); //--> false
+ *
+ * resinSemver.gte('1.16.0', 'Resin OS 2.0.2+rev2'); //--> false
+ */
+export const gte = (versionA: string | null, versionB: string | null): boolean => {
+	if (!versionA && versionB) {
+		return false;
+	}
+	if (versionA && !versionB || !versionA && !versionB) {
+		return true;
+	}
+
+	versionA = normalize(<string>versionA);
+	versionB = normalize(<string>versionB);
+
+	const isAValid = semver.valid(versionA);
+	const isBValid = semver.valid(versionB);
+
+	if (!isAValid && !isBValid) {
+		return versionA >= versionB;
+	}
+
+	if (!isAValid && isBValid) {
+		return false;
+	}
+
+	if (isAValid && !isBValid) {
+		return true;
+	}
+
+	return semver.gte(versionA, versionB);
```

Comment:
This has inconsistent behaviour with `compare`, which takes `dev` and `rev` suffixes into account. I think what should probably happen is that `gte` should be equivalent to `compare(a, b) >= 0`. Is there any reason those should give different results?